src/views/Rewards/Leaderboard.js

```diff
@@ -77,21 +77,21 @@ function TableRow({ position, account, userAccount, volume, reward }) {
 }
 
 export default function Leaderboard(props) {
-  const { weekData, userWeekData, userAccount, ensName, currentView, selectedWeek, connectWallet } = props;
+  const { weekData, userweekData, userAccount, ensName, currentView, selectedWeek, connectWallet } = props;
```

Comment:
Should it be `userweekData` or `userWeekData`? I honestly am not sure but dont mind either